src/Sylius/Behat/Context/Api/Shop/ProductReviewContext.php

```diff
@@ -75,7 +75,7 @@ public function iAddIt(): void
     public function iWantToReviewProduct(ProductInterface $product): void
     {
         $this->client->buildCreateRequest();
-        $this->client->addRequestData('productCode', $product->getCode());
+        $this->client->addRequestData('productCode', $this->iriConverter->getIriFromItem($product));
```

Comment:
```suggestion
        $this->client->addRequestData('product', $this->iriConverter->getIriFromItem($product));
```
but keeping the `productCode` on the command, could you investigate is it possible?